pkg/descheduler/policyconfig.go

```diff
@@ -140,33 +140,16 @@ func validateDeschedulerConfiguration(in api.DeschedulerPolicy, registry pluginr
 				pluginUtilities := registry[pluginConfig.Name]
```

Comment:
We need to fail in case `registry[pluginConfig.Name]` does not exists. E.g.:
```go
if _, ok := registry[pluginConfig.Name]; !ok {
	errorsInProfiles = fmt.Errorf("%w: %s", errorsInProfiles, fmt.Sprintf("in profile %s: plugin %s in pluginConfig not registered", profile.Name, pluginConfig.Name))
	continue
}
pluginUtilities := registry[pluginConfig.Name]
if pluginUtilities.PluginArgValidator == nil {
	continue
}
err := pluginUtilities.PluginArgValidator(pluginConfig.Args)
...
```
This will also reduce the level of nesting by 2.

Reply:
Good suggestion.
Update later.